### ISSUE TITLE
Draft new release with security fixes

### ISSUE THREAD

**Issue by oxzi** [dinhvh/libetpan#439]
First, thanks a lot for this library and its persistence over time. As a long time Claws Mail user, I'm using it indirectly for quite some time. Furthermore, I am maintaining a package for it on NixOS.

As a package maintainer I just found multiple security and reliability fixes in the master branch since the latest release. Without new releases, however, it is hard to timely see those. This might result in an unpatched and vulnerable software version to be delivered to the end users.

More specifically, I mean (at least) the following commits:
- https://github.com/dinhvh/libetpan/commit/720e92e5752e562723a9730f8e604cb78f3a9163
- https://github.com/dinhvh/libetpan/commit/0cdefb017fcfd0fae56a151dc14c8439a38ecc44
- https://github.com/dinhvh/libetpan/commit/68bde8b12b40a680c29d228f0b8fe4dfbf2d8d0b
- https://github.com/dinhvh/libetpan/commit/24c485495216c00076b29391591f46b61fcb3dac
- https://github.com/dinhvh/libetpan/commit/1002a0121a8f5a9aee25357769807f2c519fa50b (CVE-2020-15953)
- https://github.com/dinhvh/libetpan/commit/298460a2adaabd2f28f417a0f106cb3b68d27df9 (CVE-2020-15953)
- https://github.com/dinhvh/libetpan/commit/078b924c7f49ac435b10b0f53a73f1bbc4717064
- https://github.com/dinhvh/libetpan/commit/5c9eb6b6ba64c4eb927d7a902317410181aacbba (CVE-2022-4121)

Therefore I would kindly ask you to create a new release and do the same for upcoming security patches. Thanks!

**Comment by mones** [comment]
Yep, a new release would be nice... Any chance we could have one for Christmas @dinhvh  ? :smile: 
Happy holidays in any case!

**Comment by alex2grad** [comment]
Hi @dinhvh,

I would like to ask if it would be possible to make a new libetpan release.

The latest release (1.9.4) was published on Nov 1, 2019, and since then there have been 100+ commits merged into the master branch (currently around 104 commits).

There were also multiple important security-related fixes committed recently, in addition to many other fixes and improvements accumulated over the years.

A new release would help distributions and downstream projects consume these fixes more easily, especially the security-related ones, without depending on unreleased master snapshots.

Thank you for maintaining libetpan.

**Comment by dinhvh** [comment]
I created a new 1.10 release.
Enjoy!

https://github.com/dinhvh/libetpan/releases/tag/1.10

**Comment by oxzi** [comment]
Thanks a lot, both for the release and your maintenance as well.
